docs/en/sql-reference/window-functions/leadInFrame.md

```diff
@@ -8,11 +8,17 @@ sidebar_position: 10
 
 Returns a value evaluated at the row that is offset rows after the current row within the ordered frame.
 
+:::warning
+`leadInFrame` behavior differs from the standard SQL `lead` window function.
+Clickhouse window function `leadInFrame` respects the window frame.
+To get behavior identical to the `lead`, use `rows between unbounded preceding and unbounded following`.
```

Comment:
I will be honest, I was completely missing this construct in our SQL, thus first I couldn't understand what do you mean. By using FULL CAP text, I think it would be slightly more understandable without explaining what is this.
 
```suggestion
To get behavior identical to the `lead`, use `ROWS BETWEEN UNBOUNDED PRECEDING AND UNBOUNDED FOLLOWING`.
```

Reply:
Ahh, then SQL has much more than I have ever known!